.markdownlint.yaml

```diff
@@ -0,0 +1,260 @@
+# Default state for all rules
```

Comment:
Nice, but I have some questions:
1. What do you use as linter in vim, this here: https://github.com/fannheyward/coc-markdownlint
2. Can we create a CI target that checks this. At the moment I would make it purely optional, but in my opinion, every ruleset in iceoryx2 must be checked by CI otherwise it is not a valid rule set. Maybe we can use this here: https://github.com/igorshubovych/markdownlint-cli
3. Is there a auto-formatter?

Reply:
@elfenpiff 

1. I use: https://github.com/dense-analysis/ale
2. Yes it should be easy with https://github.com/igorshubovych/markdownlint-cli or https://github.com/DavidAnson/markdownlint-cli2 (which is supposedly faster)
3. I use: https://github.com/stevearc/conform.nvim which uses one of the CLIs on-save (2)